src/api-service/__app__/onefuzzlib/events.py

```diff
@@ -33,9 +33,53 @@ def get_events() -> Optional[str]:
         return None
 
 
+def log_event(event: Event, event_type: EventType) -> None:
+    
+    clone_event = event.copy(deep=True)
+    log_event_recurs(clone_event)
+    logging.info("sending event: %s - %s", event_type, clone_event)
+    
+
+def log_event_recurs(clone_event: Event, visited: Set[int] = set()) -> Event: 
```

Comment:
Can you add a unit test that verifies this?

Reply:
Yes, adding now.